### ISSUE TITLE
LC Review from Adrian 

### ISSUE THREAD

**Issue by danielkinguk** [ietf-ccamp-wg/draft-ietf-ccamp-flexigrid-yang#42]
tl;dr This is ready for publication with some nits that need to be
sorted out.

Best,
Adrian

===

Throughout:

Please resolve "Flexi-Grid" or "flexi-grid"

---

As a general comment, I think it was only clear to me on reading sections
6 and 7 that this YANG module is entirely made up of individual 
augmentations of leaf nodes defined in other modules. That's not a 
problem, but it might helpful to highlight this in Section 1.

---

Section 1

   This document identifies the flexi-grid components, parameters and
   their values, characterizes the features and the performances of the
   flexi-grid elements.

I think this should be

   This document identifies the flexi-grid components, parameters, and
   their values.  It characterizes the features and the performances of
   the flexi-grid elements.

---

Section 3

   RFC Editor Note: Please replace XXXX with the RFC numbers assigned to
   [RFC9093].  Please replace YYYY with the RFC number assigned to this
   document.  Please remove this note.

I suspect that the number for XXXX is 9093 :-)

Note, however, that in Section 7 you use XXXX for this document. It may
help the RFC editor if you are consistent.

---

Section 4

OLD
      We also provide nodes A
      and B identifiers, addresses and interfaces.
NEW
      We also provide the identifiers, addresses and interfaces of nodes
      A and B.
END

s/Figure 1 shows a simple topology./Figure 2 shows a simple topology./

---

5.1

s/This document aims to describe/This document describes/
s/Traffic-engineering (TE)/traffic engineering (TE)/
s/Grid provide WDM/Grid provides WDM/
s/Traffic Engineered (TE)/TE/
s/identities and groupings/identities, and groupings/
s/The figure below/Figure 3/
s/describing an Flexi-Grid/describing a Flexi-Grid/
s/specifies with/specifies the/

---

5.1

   The Flexi-Grid specific attributes in [RFC7699], including the grid
   type, channel spacing, slot width granularity, n and m parameters,
   can be used to represent the label information.

Perhaps I am being pedantic, but isn't the label a representation of 
those parameters, not the other way round?

---

5.2.  Attributes Augmentation

   There are a few characteristics augmenting to the generic TE
   topology.

I think, probably...

5.2.  Augmentations for Flexi-Grid Topology and Node

   There are a few augmentations to the generic TE topology as described
   in this section.

---

5.2

s/topology as follows./topology as shown in Figure 4./
s/is a Flexi-Grid node/is a Flexi-Grid node as shown in Figure 5./

---

5.3

/layer0/Layer 0/
s/to describe also/to also describe/

---

Section 5.3 contains a little surprise. "Here are the bandwidth 
augmentations: there are n bandwidth augmentations."

Perhaps resolve this by adding a paragraph at the very top of the 
section to say...

   No bandwidth augmentations are needed for this YANG module.

Then the rest of the section will appear as an explanation.

---

5.4

   The model augments all the occurrences of the label-restriction list
   with flexi-grid technology specific attributes using the flexi-grid-
   label-range-info grouping defined in [RFC9093].

Can you add a reference for where the label-restriction list is 
defined. I think it is RFC 8795.

---

7.

   module ietf-flexi-grid-topology {
     description
       "This module provides a YANG data model for the routing and
        wavelength assignment (RWA) Traffic Engineering (TE)
        topology in flexi-grid optical networks. The YANG model
        described in this document is a flexi-grid technology-specific
        YANG model augmenting the generic TE topology module
        (ietf-te-topology, RFC 9795) based on the RFC 7698 and 7699.

s/based on the RFC 7698 and 7699./based on RFC 7698 and RFC 7699./

---

7.

Obviously, the copyright date is broken in the YANG

**Comment by danielkinguk** [comment]
Addressed in version 12.
